source/develop/release-management/features/infra/user-portal-permissions.html.md

```diff
@@ -33,52 +33,52 @@ For each query the engine will manage `queryType`, which can be either an Admin
 
 Permission checking will be done in two levels:
 
-1.  Autorization - before executing the query: validating a user is administrator if the query is Admin query or if \`isFiltered\` is set to false.
-2.  Filtering -as part of the query execution: joining the "business logic" of the query to a permissions view, providing a single point of entry to managing permission logic.
+1.  Authorization - before executing the query: validating a user is administrator if the query is Admin query or if `isFiltered` is set to false.
```

Comment:
is administrator -> is an administrator (or: has an administrator role)
is Admin query -> is an Admin query